Dockerfile

```diff
@@ -159,7 +159,7 @@ RUN set -ex; \
 # Build podman base image
 FROM alpine:3.20 AS podmanbase
 LABEL maintainer="Max Goltzsche <max.goltzsche@gmail.com>"
-RUN apk add --no-cache tzdata ca-certificates
+RUN apk add --no-cache tzdata ca-certificates bash
```

Comment:
Hm, if everyone would add their project's dependencies into the podman base image, it would be pretty bloated quickly.

Can you not simply install `bash` into your own downstream container image?

Which Dagger action do you mean? Can you link it here along with an example GitHub Actions workflow to illustrate your problem more clearly?